Microsoft.Research/Clousot.Cache/SQLCacheModel.cs

```diff
@@ -784,5 +786,33 @@ protected override SqlConnectionStringBuilder BuildConnectionString(IClousotCach
 
       return base.BuildConnectionString(options);
     }
+
+    public static string GetDesiredDbName()
```

Comment:
But as I mentioned, you actually changed the behavior of all clients, but we want to change the behavior for only one of them. Right? 

That's why, if you're concerned about backward compatibility, extracting this logic into the upper level is better.